### ISSUE TITLE
[LiveComponent] Update parent model from children component

### ISSUE THREAD

**Issue by JJarrie** [symfony/ux#685]
Hello,

I followed the documentation about nested component (https://symfony.com/bundles/ux-live-component/current/index.html#full-embedded-component-example) and it looks like if the binding with dataModel working well in parent -> child direction, I can't make it work in child -> parent direction, the parent LiveProp aren't updated by the child component. It's normal or I'm doing something bad?

Also the child component disappear when parent is re-rendered.

My example repository: https://github.com/JJarrie/uxlivecomponentexample

Cheers!

**Comment by JJarrie** [comment]
Somes new informations. 

I can send on parent from children with something like :

```javascript
/** markdown_controller.js */
export default class extends Controller {
    async initialize() {
        this.component = await getComponent(this.element);

        this.component.on('model:set', (model, value, component) => {
            let dataModel = [];
            component.valueStore.props._attributes.dataModel.split(' ').forEach((key) => {
               const dataModelBind = key.split(':');
               dataModel[dataModelBind[0]] = dataModelBind[1];
            });
            component.getParent().set(dataModel[model], value);
        });
    }
}
```
```twig
{# markdown_textarea.html.twig #}
<div {{ attributes.add(stimulus_controller('markdown')) }} class="mb-3">
...
```
I'm still interested to know why the binding (dataModel) isn't double sided.

We can fix also the disappearing of children component with a manual id, but I don't understand why is working :

```twig
{# templates/components/edit_post.html.twig #}
<div {{ attributes }}>
    <form data-model="on(change)|*">
        <input
                type="text"
                name="post[title]"
                value="{{ post.title }}"
        >

        {{ component('markdown_textarea', {
            name: 'toto',
            dataModel: 'value:post.content',
            label: 'Content toto',
            'data-live-id': 'markdown'
        }) }}

        <a href="#"
                data-action="live#action"
                data-action-name="save"
        >Save</a>
    </form>
</div>
```

**Comment by weaverryan** [comment]
Thanks for the excellent reproducer! I've just been able to replicate - there is definitely at least a bug with the disappearing child component. I'll get back to you :)

**Comment by weaverryan** [comment]
All fixed up in #700. The only thing you would need to change in your reproducer is:

```diff
{{ component('markdown_textarea', {
    name: 'toto',
-    dataModel: 'value:post.content',
+    dataModel: 'post.content:value',
    label: 'Content toto'
}) }}
```

I'm embarrassed to admit that the server-side logic for this (and the documentation) was completely backwards. The format is `{parentPropName:childPropName}`. Or, easier, you can use `dataModel: 'post.content'`, as `value` is the default name in the child (and the PR updates the docs to show this shortest way first).

**Comment by raphaelriehl** [comment]
Hi @JJarrie, thanks for opening this issue.
I had the same bug with the child component disappearing when the parent was rendered again. I thought it was my usage that was wrong but it was indeed a bug.
I just launched a composer update just now and good news `symfony/ux-live-component` has gone to `v2.8.0`!
Problem solved thanks to your fix @weaverryan!
Thanks a lot you two !
